lib/auth-middleware/auth_middleware/__init__.py

```diff
@@ -23,6 +23,7 @@
 import requests
 from connexion.exceptions import OAuthProblem
 from flask import request, g
+from requests_oauthlib import OAuth2Session
```

Comment:
Changes to lib/ folder shouldn't be part of this PR, right? It may have got added when resolving merge conflicts. Could you please take a look?